daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Node.scala

```diff
@@ -82,12 +82,16 @@ object Node {
       stakeholders: Set[Party],
       signatories: Set[Party],
       controllers: Set[Party],
-      children: ImmArray[Nid])
+      children: ImmArray[Nid],
+      exerciseResult: Val)
```

Comment:
When present, the `Val` also needs to be visited in `GenTransaction#foldValues`.

Reply:
I was under the (maybe false) impression that because Exercises are never received over the wire but only created in the engine that we can assume that the `exerciseResult` will always be present. Am I missing something?

Reply:
[For posterity] This is the format of permanently recorded transactions; the ledger API commands to exercise are elsewhere.

Reply:
Done